mirgecom/integrators.py

```diff
@@ -45,6 +48,40 @@ def rk4_step(state, t, dt, rhs):
     return state + dt/6*(k1 + 2*k2 + 2*k3 + k4)
 
 
+def lsrk4_step(state, t, dt, rhs):
+    """Take one step using low storage 4th order Runge-Kutta."""
+    # LSERK coefficients from [Hesthaven_2008]_, Section 3.4
```

Comment:
I'd vote for making this cite part of the docstring.

Reply:
If the constants move as in the next review comment... this may be useful place to start to understand how to document them.
https://stackoverflow.com/questions/20227051/how-to-document-a-module-constant-in-python

Reply:
Would it be good enough in this case to just change the docstring to this:
```py
"""
Take one step using low storage 4th order Runge-Kutta.

This LSERK time stepping method and its coefficients are as described by
[Hesthaven_2008]_, Section 3.4
"""
```